spec/rails_helper.rb

```diff
@@ -0,0 +1,57 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'spec_helper'
+require 'rspec/rails'
+# Add additional requires below this line. Rails is not loaded until this point!
```

Comment:
@vidoseaver We can add this to rails_helper to ignore the spec folder when simplecov runs:
```ruby
require 'simplecov'
SimpleCov.start do
    add_filter "/spec/"
end
```